pulse_actions/authentication.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+class AuthenticationError(Exception):
```

Comment:
Can you rename this CredentialsError? The word Authentication might mislead users into thinking we are actually validating the credentials, when in fact we just verify they exist.